olm-catalog/serverless-operator/project.yaml

```diff
@@ -64,9 +64,9 @@ dependencies:
   operator: 1.6.0
   # Previous versions required for downgrade testing
   previous:
-    serving: 1.7.0
-    eventing: 1.7
-    eventing_kafka_broker: 1.7
+    serving: 1.8.0
```

Comment:
The version for serving needs to be `1.8`, not `1.8.0`. I thought the "z" version would be stripped but it remains there.